vendor/github.com/cpuguy83/go-md2man/v2/md2man/roff.go

```diff
@@ -233,7 +233,7 @@ func (r *roffRenderer) handleItem(w io.Writer, node *blackfriday.Node, entering
 		} else if node.ListFlags&blackfriday.ListTypeDefinition != 0 {
 			// state machine for handling terms and following definitions
 			// since blackfriday does not distinguish them properly, nor
-			// does it seperate them into separate lists as it should
```

Comment:
Please revert the change in this file as it is vendored code which is dropped in via go mod vendor.